### DISCUSSION TITLE
Changing BPM in the editor doesn't reload timing markers on the playfield

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#17094] [Q&A] [answered]
Version 2022.226.0 | taiko and mania gamemodes
This one is hard to put into words. If you enter the editor and change a timing point, the ticks in the timeline at the top do update while the "barlines" (at least that's what they're called in taiko) on the playfield do not, leading to misleading behavior. In both screenshots, I've aligned myself to a new "barline" tick, but that's not visually represented in the playfield.
![osu_2022-03-03_23-55-08](https://user-images.githubusercontent.com/74262407/156723778-23bee53a-aaac-4b52-a89e-f4bb96c9ab86.png)
![osu_2022-03-03_23-59-47](https://user-images.githubusercontent.com/74262407/156723774-e3e73c0c-460b-4a17-a092-8d4d8c2eaf7e.png)
When saving and reloading, everything displays visually how it should.
It should also be noted that small changed to the Scroll Speed (less than 0.5) are also not represented properly on the playfield until saving and reloading the beatmap. This makes mapping in these gamemodes especially difficult.

**Comment by bdach** [comment]
tangentially aware of this, was mentioned in https://github.com/ppy/osu/pull/16571

**Comment by bdach** [comment, marked as answer]
escalated to #17127
